src/corehost/cli/deps_resolver.cpp

```diff
@@ -486,6 +486,16 @@ void deps_resolver_t::resolve_probe_dirs(
     {
         if (probe_entry_in_configs(entry, &candidate))
         {
+            // For standalone apps, on win7, coreclr needs ApiSets which has to be in the DLL search path.
+            if (!m_portable && pal::need_api_sets()
```

Comment:
Policy does loadlibrary on coreclr, and brings in all these dependencies and it also processes the deps file.